### PR TITLE
Add torrent clients and media player templates along with classes for them

### PR DESCRIPTION
**Pull request by Dyras**
Hello! This adds qBittorrent as a default template, along with a class for it to fit into since "Download Manager" didn't really fit the bill.

### CODE REVIEW

**Review by isaak654 — CHANGES_REQUESTED** [review]
1) In general, I think it would be better to merge these changes to the previous pull request, in order to avoid potential conflicts: https://github.com/sandboxie-plus/Sandboxie/pull/708

2) You forgot to add the class in this file: https://github.com/sandboxie-plus/Sandboxie/blob/cbb22e1adb5a87370b3cfe4fdfc87c33c6ba611a/Sandboxie/apps/control/ThirdPartyDialog.cpp#L819

**Comment by Dyras** [comment]
Well crap.

My plan was actually to wait with this one until the other one is merged, then I change it to remove the conflicts.

**Comment by isaak654** [comment]
> My plan was actually to wait with this one until the other one is merged, then I change it to remove the conflicts.

If this was your intention, next time you could consider to open a pull request in draft mode. I think it's more useful in these cases, because it prevents a commit to be merged accidentally: https://github.blog/2019-02-14-introducing-draft-pull-requests

**Comment by Dyras** [comment]
Sorry, I'm new when it comes to pull requests and GitHub usage in general. I'll use draft mode next time, but for now I think I don't think any major changes will be necessary.

Planning on adding Deluge and BiglyBT later today. PicoTorrent already has a finished template, but has this issue making it virtually useless: https://github.com/sandboxie-plus/Sandboxie/issues/720

File: Sandboxie/msgs/Sbie-English-1033.txt [hunk 1]
```diff
@@ -3288,6 +3288,22 @@ Email Reader
...
 .
 
+4393;txt;01
+Media Players
+.
+
+4394;txt;01
+The following exclusions allow Media players running in this sandbox to access files outside the sandbox.
+.
+
```
Comment (isaak654) [inline Sandboxie/msgs/Sbie-English-1033.txt:3298]: You missed this from the old pull request:
```
4395;txt;01
Allow %2 direct access to the Photo folder for easier screen capture.
.
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: Sandboxie/msgs/Sbie-English-1033.txt [hunk 2]
```diff
@@ -3288,6 +3288,22 @@ Email Reader
...
+4396;txt;01
+Torrent Clients
+.
+
+4397;txt;01
+The following exclusions allow Torrent clients running in this sandbox access to files outside the sandbox.
```
Comment (isaak654) [inline Sandboxie/msgs/Sbie-English-1033.txt:3304]: Replace it with:
`The following exclusions allow Torrent clients running in this sandbox to access files outside the sandbox.`

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Pull request merged**